lib/std/crypto/sha3.zig

```diff
@@ -182,6 +189,16 @@ fn ShakeLike(comptime security_level: u11, comptime delim: u8, comptime rounds:
             self.st.st.clear(0, State.rate);
         }
 
+        /// Align the input to a block boundary.
+        pub fn fillBlock(self: *Self) void {
+            self.st.fillBlock();
+        }
+
+        /// Return the KMAC function with the same security level.
+        pub fn toMac() type {
```

Comment:
This really confused me initially.

I'm not sure what the general design guidance is on parameterless type-returning functions (do we want/need them at all?), but I think this should at least be PascalCase.
The doc comment is a little confusing too, since "function" is an overloaded term here and it could be interpreted as either "SHA-3 derived function", or literally as returning a Zig `fn` type. Maybe "instance" is a slightly better word here? I don't know.

(Thanks for working on this btw)

Reply:
Thanks for your feedback!

The function should indeed be PascalCase.

That has been done, and the comments have been improved.

Thanks!